src/linter/messages.ts

```diff
@@ -333,6 +334,15 @@ export const MESSAGE_INFO = {
 		details: () => `"{@link sap.ui.core.RenderManager#methods/icon RenderManager}",`,
 	},
 
+	[MESSAGE.NOT_STATIC_CONTROL_RENDERER]: {
+		severity: LintMessageSeverity.Warning,
+		ruleId: RULES["no-deprecated-api"],
+
+		message: ({className}: {className?: string}) =>
+			`The control renderer ${className ? ("of '" + className + "'") : ""} must be a static property`,
```

Comment:
There will be two spaces in case `className` is not provided.

Reply:
Done